actionpack/lib/action_dispatch/journey/route.rb

```diff
@@ -51,13 +51,13 @@ def self.verb_matcher(verb)
 
       def self.build(name, app, path, constraints, required_defaults, defaults)
         request_method_match = verb_matcher(constraints.delete(:request_method))
-        new name, app, path, constraints, required_defaults, defaults, request_method_match, 0
+        new name, app, path, constraints, required_defaults, defaults, request_method_match, 0, {}
```

Comment:
I think we can safely assume `{}` by default for `scope_options `. Also, I think we're far past the point of having too many positional arguments for this constructor (and `Mapping`'s constructor). We should look at using keyword args instead, but that can be done in another PR. They both seem to be private API so we don't need deprecation cycles.